src/methods/modular_programs/medium/q9/MaxItOut.java

```diff
@@ -0,0 +1,46 @@
+package methods.modular_programs.medium.q9;
+
+/*
+Write a program that asks for two integers and passes them to a method called maxOut which check is one of the numbers
+is divisible by the other. If so, it returns a string of the format "x is the max". If not, it should return “Not
+divisible”. The main method then prints the returned string.
+
+Sample Output:
+  Enter the first number:
+  10
+  Enter the second number:
+  5
+  10 is the max
+ */
+
+import java.util.Scanner;
+
+public class MaxItOut {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Enter the first number: ");
+        int n1 = input.nextInt();
+        System.out.println("Enter the second number: ");
+        int n2 = input.nextInt();
+        System.out.println(maxOut(n1, n2));
+    }
+
+    public static String maxOut(int n1, int n2) {
+        String res;
+        if (n1 % n2 == 0) {
+            if (n1 < n2)
+                res = n2 + " is the max";
+            else
```

Comment:
I don't think you are ever going to need the else, because if a number is divisible by another number, the divisor must be less than the dividend. So if n1 % n2 == 0, then n1 is the max

Reply:
So you should be able to simplify the solution code

Reply:
unfortunately, when one of the cases is 0 we do need the if statement